lib/services/local/plugins.go

```diff
@@ -74,7 +77,7 @@ func (s *PluginsService) DeletePlugin(ctx context.Context, name string) error {
 // DeleteAllPlugins implements service.Plugins
 func (s *PluginsService) DeleteAllPlugins(ctx context.Context) error {
 	startKey := backend.Key(pluginsPrefix)
-	err := s.backend.DeleteRange(ctx, startKey, backend.RangeEnd(startKey))
+	err := s.backend().DeleteRange(ctx, startKey, backend.RangeEnd(startKey))
```

Comment:
Just in case we end up adding a resource that's just a singleton called `/plugint` (😅), this should use `startKey := backend.Key(pluginsPrefix, "")`